config/config.go

```diff
@@ -31,6 +31,19 @@ type ArcConfig struct {
 	Cache              *CacheConfig        `mapstructure:"cache"`
 }
 
+type BroadcastingConfig struct {
+	Mode           string        `mapstructure:"mode"`
+	Ipv6Enabled    bool          `mapstructure:"ipv6Enabled"`
+	MulticastGroup string        `mapstructure:"multicastGroup"`
```

Comment:
I'm not an expert on IPv6, so it's just a question out of curiosity, what is this multicastGroup for? Is it the group that Arc is a part of, or is it the group that Arc will be broadcasting to?

And is it enough to always have only one group?

Reply:
@shotasilagadzetaal 
After a meeting with Jake & Yurii we found that `MulticastGroup` needs to be a list as there can be multiple multicast groups